Formula/p/purr.rb

```diff
@@ -17,9 +17,17 @@ class Purr < Formula
   end
 
   depends_on "fzf"
+
+  on_macos do
+    depends_on "gnu-sed" => :build
+  end
+
   uses_from_macos "zsh"
 
   def install
+    # For `sed -i` usage used to remove comments
+    ENV.prepend_path "PATH", Formula["gnu-sed"].libexec/"gnubin" if OS.mac?
```

Comment:
Given this is just in the Makefile, may be worth seeing if upstream will accept a patch to fix this.

Reply:
Can look into afterward (will add to personal backlog). For now want to get more `all` bottles built prior to our Sequoia bottling.